### ISSUE TITLE
consider pip install instead of cloning

### ISSUE THREAD

**Issue by mathisloevenich** [psyinfra/HTCrystalBall#26]
Till now your instructions are:
```
clone this repo to the file system of any machine in your HTCondor cluster

git clone git@github.com:psyinfra/HTCrystalBall.git

cd into the newly created directory and install python modules using the setup.py

cd HTCrystalBall

pip3 install .
```

I know from my project that it is possible to make all needed files be installed by something like
- pip install git+https://github.com/psyinfra/HTCrystalBall.git   

This would make it easier to just get the files you need, as you won't need the README and requirements.txt etc.

Also it reduces to just one simple command

**Comment by aqw** [comment]
I like the suggestion. Any chance you'd be willing to open a PR with this change?

**Comment by jmktf** [comment]
@mathisloevenich would you mind changing this and making a PR so that I can review it and apply this change?

**Comment by mathisloevenich** [comment]
> @mathisloevenich would you mind changing this and making a PR so that I can review it and apply this change?

Yes I can do it, but it will need changes on the setup.py and usually this has to be pushed on master, to then fully test it. 
I can write the code, but you are the on that needs to test it, if it's working.

**Comment by aqw** [comment]
@mathisloevenich `pip` does support installing from non-master branches (e.g. `pip install git+https://github.com/psyinfra/HTCrystalBall.git@other_branch`). But if you want to avoid the syntax gymnastics, you can have your work in the `master` branch of your fork. Then you can easily test your changes.

You're right though, that Travis CI might struggle. I'm unsure if that tests the install procedure, and how that's setup. But that's alright.

**Comment by mathisloevenich** [comment]
PR done.

**Comment by mathisloevenich** [comment]
I made critical changes to the script, that might be not wanted, so please only consider the PR as a guide.
Because my script is executable by : 
- htcompact -h

I also made this change to your script so it will be executable like:
- htcrystalball -h

You don't have to adopt this.

**Comment by nhjjreuter** [comment]
This issue has been addressed in PR #36 and can be closed. It is now possible to run HTCrystalBall using either `htcb` or `htcrystalball` with the subcommands `peek` and `configure` (the latter of which is not implemented and may not be used at all depending on the state of the slot configuration, in which case the subcommands should be removed).
